### PR TITLE
More explicit comment on switch in registerDelay

### PR DESCRIPTION
**Pull request by qrilka**

### CODE REVIEW

**Comment by bgamari** [comment]
Thanks!

File: libraries/base/GHC/Conc/IO.hs [hunk 1]
```diff
@@ -188,8 +188,9 @@ threadDelay time
         case delay# time# s of { s' -> (# s', () #)
         }}
 
--- | Set the value of returned TVar to True after a given number of
--- microseconds. The caveats associated with threadDelay also apply.
+-- | Switch the value of returned 'TVar' from intial value 'False' to 'True'
```
Comment (utdemir) [inline libraries/base/GHC/Conc/IO.hs]: `s/intial/initial/`

Comment (qrilka) [reply libraries/base/GHC/Conc/IO.hs]: Oops, fixed

**Comment by bgamari** [comment]
Merged with 1154c9b6332342e0447ab64c082b016862bc97da.

**Pull request closed without merging**